src/Internal/Deserialization/Transaction.purs

```diff
@@ -473,31 +472,27 @@ convertProtocolParamUpdate cslPpu = do
   let
     ppu = _unpackProtocolParamUpdate maybeFfiHelper cslPpu
     lbl = (<>) "ProtocolParamUpdate."
-
-  minfeeA <- traverse (map Coin <<< BigNum.toBigInt' (lbl "minfeeA"))
-    ppu.minfeeA
-  minfeeB <- traverse (map Coin <<< BigNum.toBigInt' (lbl "minfeeB"))
-    ppu.minfeeB
+    minfeeA = map (Coin <<< BigNum.toBigInt) ppu.minfeeA
+    minfeeB = map (Coin <<< BigNum.toBigInt) ppu.minfeeB
   maxBlockBodySize <- traverse (cslNumberToUInt (lbl "maxBlockBodySize"))
     ppu.maxBlockBodySize
   maxTxSize <- traverse (cslNumberToUInt (lbl "maxTxSize")) ppu.maxTxSize
   maxBlockHeaderSize <- traverse (cslNumberToUInt (lbl "maxBlockHeaderSize"))
     ppu.maxBlockHeaderSize
-  keyDeposit <- traverse (map Coin <<< BigNum.toBigInt' (lbl "keyDeposit"))
-    ppu.keyDeposit
-  poolDeposit <- traverse (map Coin <<< BigNum.toBigInt' (lbl "poolDeposit"))
-    ppu.poolDeposit
+  let
+    keyDeposit = map (Coin <<< BigNum.toBigInt) ppu.keyDeposit
+    poolDeposit = map (Coin <<< BigNum.toBigInt) ppu.poolDeposit
```

Comment:
Same as the other suggestion, `<$>`

Reply:
Why <$> is better?

Reply:
It's not significantly better, but you only tend to see map used when the operator would require parenthesis, or when being partially applied

```suggestion
  let
    keyDeposit = Coin <<< BigNum.toBigInt <$> ppu.keyDeposit
    poolDeposit = Coin <<< BigNum.toBigInt <$> ppu.poolDeposit
```